### PR TITLE
Various improvements & bug fixes

### PR DESCRIPTION
**Pull request by hburn7**
- Submitter is now properly displayed in the admin panel
- Updated some routes to use the API client
- Updated some methods to use concrete types instead of anonymous types.

Sorry about some weird formatting, idk why my laptop does this.

### CODE REVIEW

File: components/Tournaments/MainExpandableRow/MainExpandableRow.tsx [hunk 1]
```diff
@@ -15,13 +15,14 @@ import InfoContainer from '../InfoContainer/InfoContainer';
...
+import { TournamentDTO } from '@osu-tournament-rating/otr-api-client';
 
-export default function MainExpandableRow({ tournament }: { tournament: {} }) {
+export default function MainExpandableRow({ tournament }: { tournament: TournamentDTO }) {
   const [expanded, setExpanded] = useState(false);
   const [fetchedData, setFetchedData] = useState(null);
   const [fetchLoading, setFetchLoading] = useState(false);
 
-  const format = `${tournament?.lobbySize}v${tournament?.lobbySize}`;
+  const lobby_size = `${tournament?.lobbySize}v${tournament?.lobbySize}`;
```
Comment (myssto) [inline components/Tournaments/MainExpandableRow/MainExpandableRow.tsx:25]:
```suggestion
  const lobbySize = `${tournament?.lobbySize}v${tournament?.lobbySize}`;
```

File: components/Tournaments/MainExpandableRow/MainExpandableRow.tsx [hunk 2]
```diff
@@ -65,7 +66,7 @@ export default function MainExpandableRow({ tournament }: { tournament: {} }) {
                 {tournament.name}
               </motion.span>
             </div>
-            <span>{format}</span>
+            <span>{lobby_size}</span>
```
Comment (myssto) [inline components/Tournaments/MainExpandableRow/MainExpandableRow.tsx:69]:
```suggestion
            <span>{lobbySize}</span>
```

File: components/Tournaments/MainExpandableRow/MainExpandableRow.tsx [hunk 3]
```diff
@@ -90,7 +91,7 @@ export default function MainExpandableRow({ tournament }: { tournament: {} }) {
                 )}
               </div>
             </span>
-            <span>Missing Submitter</span>
+            <span>{tournament.submittedByUser?.player.username}</span>
```
Comment (myssto) [inline components/Tournaments/MainExpandableRow/MainExpandableRow.tsx:94]:
```suggestion
            <span>{tournament.submittedByUser?.player.username ?? 'Missing Submitter'}</span>
```
This still has a chance to be null

File: app/actions.ts [hunk 4]
```diff
@@ -281,43 +281,35 @@ export async function fetchTournamentsPage(params: {}) {
   return data.result;
 }
 
-export async function fetchTournamentPage(tournament: string | number) {
+export async function fetchTournamentPage(tournament_id: number | string) {
```
Comment (myssto) [inline app/actions.ts:284]:
```suggestion
export async function fetchTournamentPage(tournamentId: number | string) {
```

File: app/actions.ts [hunk 5]
```diff
@@ -281,43 +281,35 @@ export async function fetchTournamentsPage(params: {}) {
...
-        'Content-Type': 'application/json',
-        Authorization: `Bearer ${session.accessToken}`,
-      },
-    }
-  );
+  const wrapper = new TournamentsWrapper(apiWrapperConfiguration);
 
-  data = await data.json();
+  let data = await wrapper.get({
+    id: tournament_id as number,
```
Comment (myssto) [inline app/actions.ts:293]:
```suggestion
    id: tournamentId as number,
```

File: app/actions.ts [hunk 6]
```diff
@@ -281,43 +281,35 @@ export async function fetchTournamentsPage(params: {}) {
...
+    verified: false
+  })
 
-  return data;
+  return data.result;
 }
 
-export async function fetchMatchPage(match: string | number) {
+export async function fetchMatchPage(match_id: string | number) {
```
Comment (myssto) [inline app/actions.ts:300]:
```suggestion
export async function fetchMatchPage(matchId: string | number) {
```

File: app/actions.ts [hunk 7]
```diff
@@ -281,43 +281,35 @@ export async function fetchTournamentsPage(params: {}) {
...
-    headers: {
-      'Content-Type': 'application/json',
-      Authorization: `Bearer ${session.accessToken}`,
-    },
-  });
+  const wrapper = new MatchesWrapper(apiWrapperConfiguration);
 
-  data = await data.json();
+  let data = await wrapper.get({
+    id: match_id as number,
```
Comment (myssto) [inline app/actions.ts:309]:
```suggestion
    id: matchId as number,
```

**Review by myssto — CHANGES_REQUESTED** [review]
this isnt python cmon 🤓

**Comment by hburn7** [comment]
@myssto I'll let you since the conflicts since I PR'd before you pushed :')

**Review by AkinariHex — APPROVED** [review]

**Pull request merged**